core/tests/unit/api_core/gapic/test_method.py

```diff
@@ -97,6 +97,26 @@ def test_wrap_method_with_default_retry_and_timeout(unusued_sleep):
 
 
 @mock.patch('time.sleep')
+def test_wrap_method_with_default_retry_and_timeout_using_sentinel(
+        unusued_sleep):
+    method = mock.Mock(spec=['__call__'], side_effect=[
+        exceptions.InternalServerError(None),
+        42])
```

Comment:
This indenting disturbs my sensibilities for some reason. What about?

```python
method = mock.Mock(
    spec=['__call__'],
    side_effect=[
        exceptions.InternalServerError(None),
        42,
    ],
)
```

Reply:
Done.